package.json

```diff
@@ -242,8 +242,16 @@
       "@commitlint/config-conventional"
     ],
     "rules": {
-      "header-max-length": [2, "always", 50],
-      "body-max-line-length": [2, "always", 72]
+      "header-max-length": [
```

Comment:
just wondering, did we change our linter settings?

Reply:
Might be from a prettier update, the change happened with the linter/formatting. Did you not get the same change?

Reply:
🤔not sure where it came from, just noticed that it was causing merge conflicts